docs/api/ref/schemas/product_base.yaml

```
@@ -0,0 +1,80 @@
+type: object
+description: |
+  Base product data
+properties:
+  abbreviated_product_name:
+    type: string
+    description: Abbreviated name in requested language
+  code:
+    type: string
+    description: |
+      barcode of the product (can be EAN-13 or internal codes for some food stores),
+      for products without a barcode,
+      Open Food Facts assigns a number starting with the 200 reserved prefix
+  codes_tags:
+    type: array
+    items:
+      type: string
+      description: |
+        A value which is the type of barcode "code-13" or "code-8"
+        and
+        A series of mask for the barcode
+        It helps retrieve barcodes starting by
+      example:
+        ["code-13","3017620422xxx","301762042xxxx","30176204xxxxx","3017620xxxxxx","301762xxxxxxx","30176xxxxxxxx","3017xxxxxxxxx","301xxxxxxxxxx","30xxxxxxxxxxx","3xxxxxxxxxxxx"]
+  generic_name:
+    type: string
+    description: |
+      Legal name of the product as regulated
+      by the European authorities.
+  id:
+    description: alias of `code`
```

```suggestion
    description: internal identifier for the product, usually set to the value of `code`, except on the producers platform where it is prefixed by the owner
```

e.g. _id: "org-arcadie/3417960044406",